docs/tutorials/multimodal/ner.md

```diff
@@ -0,0 +1,76 @@
+# AutoMM for Named Entity Recognition
+
+Named entity recognition (NER) refers to identifying and categorizing key information (entities) from unstructured text. An entity can be a word or a series of words which correspond to categories such as cities, time expressions, monetary values, facilities, person, organization, etc. An NER model usually takes as input an unannotated block of text and output an annotated block of text that highlights the named entities with predefined categories. For example, given the following sentences, 
+
+- Albert Einstein was born in Germany and is widely acknowledged to be one of the greatest physicists.
+
+The model will tell you that "Albert Einstein" is a PERSON and "Germany" is a LOCATION. In the following, we will introduce how to use AutoMM for the NER task, including how to prepare your data, how to train your model, and what you can expect from the model outputs.
+
+ 
+## Data Preparation
+Like other tasks in AutoMM, all you need to do is to prepare your data as data tables (i.e., dataframes) which contain a text column and an annotation column. The text column stores the raw textual data which contains the entities you want to identify. Correspondingly, the annotation column stores the label information (e.g., the *category* and the *start/end* offset in character level) for the entities. AutoMM requires the annotation column to have the following json format. 
```

Comment:
We can consider to highlight the `annotation column` because it's a new concept, like 

```
*annotation* column.
```

Reply:
addressed in [commit](https://github.com/awslabs/autogluon/pull/2232/commits/c71ce038499bfff90493cbb9afac904b2df5ca0a)